examples/undocumented/python/classifier_perceptron.py

```diff
@@ -1,42 +0,0 @@
-#!/usr/bin/env python
```

Comment:
let's name the file consistently with the others... just `perceptron.sg` (and same for the data)